.travis.yml

```diff
@@ -2,7 +2,10 @@ language: node_js
 node_js:
   - "0.10"
   - "0.12"
-  - "iojs"
+  - "1"
+  - "2"
+  - "3"
+  - "4"
```

Comment:
Don't think travis supports all of these versions

http://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Provided-Node.js-Versions